app/src/main/java/net/schueller/peertube/service/VideoPlayerService.java

```diff
@@ -178,11 +178,18 @@ public void setCurrentStreamUrl(String streamUrl)
 
     //Playback speed control
     public void setPlayBackSpeed(float speed) {
-
         Log.v(TAG, "setPlayBackSpeed...");
         player.setPlaybackParameters(new PlaybackParameters(speed));
     }
 
+    /**
+     * Returns the current playback speed of the player.
+     * @return the current playback speed of the player.
+     */
```

Comment:
Should I add a javadoc `@since` tag here?

Reply:
Not needed, we are still in beta.